### ISSUE TITLE
Length conversion should use Feet + Inches instead of Feet with decimal point

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#57]
**Problem Statement**
Today, length converter shows decimal values for feet, while in practice, feet + inches are oftentimes used.

**Evidence or User Insights**
Observation of current behavior and customary use in U.S. and U.K. Details on [foot (unit)](https://simple.wikipedia.org/wiki/Foot_(unit)).

**Proposal**
When converting to feet, in addition to showing the decimal conversion, also include "feet + inches", which is more commonly used when working in this unit. For example, when converting 230cm, Calculator shows 7.545932 feet. It would be more useful if it would _also_ show 7' 6.55" (feet with inches).

Possibly dependent on #379.

**Goals**
* Additionally show the "feet and inches" conversion when working with feet in length converter.

**Non-Goals**
* Remove decimal representation of values in feet

**Low-Fidelity Concept**
![image](https://user-images.githubusercontent.com/1414423/63055635-329eff80-be9b-11e9-850b-a50a15eab2e9.png)

**Requested Assignment**
I'm just suggesting this idea.  I don't want to implement it.

**Comment by aaaxx** [comment]
Ideally, it should show both, the non-default notation probably in the "About equal to" section.
